src/components/Dashboard/Github/dashboard.tsx

```diff
@@ -207,6 +212,32 @@ const DashboardGithub: React.FunctionComponent = () => {
             </ModalBody>
           </Modal>
         )}
+        {deleteDraftContribution != undefined && (
+          <Modal
+            isOpen
+            title="Delete contribution"
```

Comment:
Don't use `title` when also using a `<ModalHeader>`. All that it does is add a tooltip showing the same information.